docs/make.jl

```diff
@@ -1,4 +1,7 @@
 cd(@__DIR__)
+using Pkg; Pkg.activate(@__DIR__)
+const CI = get(ENV, "CI", nothing) == "true"
+CI && (ENV["GKSwstype"] = "100")

```

Comment:
Out of curiosity, what is GKSwstype ?

Reply:
No clue, its so using GR works on Travis...

Reply:
Fair enough.